CHANGELOG.md

```diff
@@ -1,12 +1,14 @@
 # Rockset Go Client Changelog
 
+## v0.8.0 2020-03-03
```

Comment:
The version you defined in `rockclient.go` is `0.9.0`

Reply:
nice good catch, updated